### PR TITLE
feat: update start and run commands to use bubble tea styling

### LINKED ISSUE #508: cli: improve start command DX

### LINKED ISSUE #509: cli: improve run command DX

### PR DESCRIPTION
**Pull request by HomelessDinosaur**
Closes https://github.com/nitrictech/nitric/issues/509
Closes https://github.com/nitrictech/nitric/issues/508

### CODE REVIEW

**Comment by jyecusch** [comment]
I'm seeing some odd behaviour when resizing the terminal

resize

**Review by tjholm — DISMISSED** [review]

**Comment by davemooreuws** [comment]
I can see two quit messages at end of `nitric run`

image

**Comment by davemooreuws** [comment]
On `nitric start` Don't start browser until a stack is found (see screenshot). Like the current behavior. 
image

File: pkg/tui/localservices/localservices.go [hunk 1]
```diff
@@ -0,0 +1,356 @@
...
+	// Create a debouncer for the refresh and remove locking
+	debounced := debounce.New(500 * time.Millisecond)
+
+	dashboardStarted := false
+
+	// React to worker pool state and update services table
+	pool.Listen(func(we run.WorkerEvent) {
+		// Serve the dashboard and open up the browser
+		if !dashboardStarted {
+			err = dash.Serve(ls.GetStorageService(), noBrowser || output.CI)
```
Comment (davemooreuws) [inline pkg/tui/localservices/localservices.go]: This runs twice on the run command now. Causing an error.

File: pkg/tui/buildlog/help.go [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+		key.WithKeys("up"),
+		key.WithHelp("↑", "scroll up"),
+	),
+	Down: key.NewBinding(
+		key.WithKeys("down"),
+		key.WithHelp("↓", "scroll down"),
+	),
+	Quit: key.NewBinding(
+		key.WithKeys("q", "ctrl+c"),
+		key.WithHelp("q", "quit"),
```
Comment (davemooreuws) [inline pkg/tui/buildlog/help.go]: shall we change quit to esc to mirror the new project command?

**Review by davemooreuws — CHANGES_REQUESTED** [review]

**Review by davemooreuws — APPROVED** [review]

**Comment by davemooreuws** [comment]
Approved. Lets take note of persisting the build log for v1/new PR

**Review by tjholm — APPROVED** [review]

**Pull request merged**